File: C/Exercicio14.c
#include <stdio.h>
 
int main(int argc, char *argv[]){
  float A,B,C;
  
  printf("Digite o primeiro numero:");
  scanf("%f", &A);
 
  printf("Digite o segundo numero: ");
  scanf("%f", &B);

  printf("Digite o terceiro numero: ");
  scanf("%f", &C);
 


  if (A>B && A>C)
   {
      printf("O maior numero e: %1.f", A);
  }

   else if (B>C && B>A) {

       printf("O maior numero e: %1.f",B);
  }

  else {
      
       printf("O maior numero e: %1.f",C);
       }
     
     

  return 0;
}

File: Python/Exercicio14.py
a = float(input('Digite o primeiro valor: '))
b = float(input('Digite o segundo valor:'))
c = float(input('Digite o terceiro valor:'))

 
if a > b and a > c:
     print(f"O maior número é {a:.2f}")
elif b > a and b >c :
     print(f"O maior número é {b:.2f}")
else:
    print(f"O maior número é {c:.2f}")